plugins/oidc-login.yaml

```diff
@@ -3,30 +3,20 @@ kind: Plugin
 metadata:
   name: oidc-login
 spec:
-  shortDescription: Login for OpenID Connect authentication
+  homepage: https://github.com/int128/kubelogin
+  shortDescription: kubectl integration for OpenID Connect authentication
   description: |
-    This plugin gets a token from the OIDC provider and writes it to the kubeconfig.
```

Comment:
I found the previous description much better, I have to say. Is there a particular reason why you were unhappy with it?

Reply:
Since v1.14.0, kubelogin can be run as a client-go credential plugin or a standalone command. So the previous description is not accurate. I will revise the description.